packages/workbox-webpack-plugin/package.json

```diff
@@ -23,7 +23,8 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
-    "workbox-build": "^2.0.1"
+    "workbox-build": "^2.0.1",
+    "workbox-sw": "^2.0.1"
   },
   "author": "Google's Web DevRel Team",
```

Comment:
Feel free to add your name!

Reply:
Yeah. We definitely want you to feel like as much a part of the team as anyone else. Feel free to add yourself :)

@jeffposnick @gauntface at some point, we might want to change this to "Workbox Core Team" or reference an owners list we can also add folks like Will to.